boards/nxp/imx91_evk/board.yml

```diff
@@ -1,6 +1,6 @@
 board:
   name: imx91_evk
-  full_name: i.MX91 EVK
+  full_name: i.MX91-EVK
```

Comment:
```suggestion
  full_name: i.MX91EVK
```
See https://www.nxp.com/design/design-center/development-boards-and-designs/i.MX91EVK

Reply:
Not critical, but "i.MX91-EVK" is also used by nxp.com
![image](https://github.com/user-attachments/assets/771300df-65bc-471f-a968-834683242970)